robotino_description/urdf/base.urdf.xacro

```diff
@@ -5,22 +5,86 @@
 
   <xacro:macro name="base" params="name">
   
-    <!--link name="${name}_footprint">
-   <visual>
+   <joint name="origin_joint" type="prismatic">
```

Comment:
@shangl I can call you but please also write here the explanation, it will be saved on the github history, in case that someone needs to modify this line on the future, can check the github history and know why was added this line.